### DISCUSSION TITLE
workflow for a new training

### DISCUSSION THREAD

**Discussion by oscarramirezs** [Project-MONAI/tutorials#1236] [Q&A]
maybe it's a bit of a silly question; but I'm new on this. 

I trained a Unet model with 88 patients, which works relatively well (the training was long, because I don't have a GPU).
Right now I have another 72 new patients, and I want to re-traing the model, but I don't know the best way,
1. launch a new learning with 150 patients (the total of my patients, It will be really long) 
2. launch a new training with only the 82 new cases? (won't the model forget the old cases?)
3. Put few harder old cases with all the new ones?
4. Do I need to train the same trained model? or do i need to do like transfer learning and randomize last parameters..(such as Model Zoo tutorial) 

thank for your help, I will appreciate it a lot

Oskar

**Comment by KumoLiu** [comment]
Hi @oscarramirezs, thank you for your interest here.
When you have some new data:

- First, you could certainly re-train all the data. 
- If you only retrain on new data for the second method you mentioned, it is equivalent to not using the first part of the data.
- For the third way, I don't think it's necessary, but you could certainly try it.
But I noticed that you said you don't have a GPU and the model with 88 patients is already doing well, I think you could use the weight from the first part and retrain it on the second part. If the result is not very well, then you could train all the data from scratch.

Hope it can help you, thanks!
